### PR TITLE
feat: add localize for ES, EN translations

### PR DESCRIPTION
**Pull request by icoliz**
- remove constructor
- fix unit test filter event
- update visual test name
- add localize in Spanish and English

### CODE REVIEW

File: packages/villagers-filters/src/VillagersFilters.js [hunk 1]
```diff
@@ -53,11 +62,16 @@ export class VillagersFilters extends ScopedElementsMixin(LitElement) {
           <lion-input
             class="search-input"
             name="search-input"
-            placeholder="Example: Cheri"
-            label="Villager name"
+            data-testid="search-input"
+            placeholder=${localize.msg(
+              'villagers-filters:searchInputPlaceholder'
+            )}
+            label=${localize.msg('villagers-filters:searchInputLabel')}
```
Comment (carlosmmdiaz) [inline packages/villagers-filters/src/VillagersFilters.js]: You should use LOCALE_KEY

File: packages/villagers-filters/test/VillagersFilters.test.js [hunk 2]
```diff
@@ -14,11 +14,16 @@ describe('VillagersFilters', () => {
   });
 
   // Review this test
-  it('should dispatch click-search-button event with the search', async () => {
+  it.only('should dispatch click-search-button event with the search', async () => {
```
Comment (carlosmmdiaz) [inline packages/villagers-filters/test/VillagersFilters.test.js]: Remove only please

**Comment by icoliz** [comment]
- Used LOCALE_KEY variable
- removed '.only' and unused waitUntil from test file
- removed unused variable

**Comment by icoliz** [comment]
Changed button color to fix accessibility issues

File: packages/villagers-filters/src/VillagersFilters.styles.js [hunk 3]
```diff
@@ -9,7 +9,6 @@ export const styles = css`
   :host {
     --main-dark: #619c86;
```
Comment (carlosmmdiaz) [inline packages/villagers-filters/src/VillagersFilters.styles.js]: Delete unused variables please

Comment (icoliz) [reply packages/villagers-filters/src/VillagersFilters.styles.js]: Done

**Pull request merged**